src/liballoc/arc.rs

```diff
@@ -577,9 +585,7 @@ impl<T: ?Sized> Drop for Arc<T> {
         // [1]: (www.boost.org/doc/libs/1_55_0/doc/html/atomic/usage_examples.html)
         atomic::fence(Acquire);
 
-        unsafe {
-            self.drop_slow()
-        }
+        unsafe { self.drop_slow() }
```

Comment:
Why is this one turned into one-line, but the previous unsafe block has the precise opposite?